src/SimulationRunner.cc

```diff
@@ -463,6 +463,15 @@ void SimulationRunner::PublishStats()
 
   msg.set_paused(this->currentInfo.paused);
 
+  msg.mutable_step_size();
+  if (!this->currentInfo.paused)
+  {
+    auto stepSizeSecNsec =
+      math::durationToSecNsec(this->stepSize);
+    msg.mutable_step_size()->set_sec(stepSizeSecNsec.first);
+    msg.mutable_step_size()->set_nsec(stepSizeSecNsec.second);
```

Comment:
Should be able to use `msgs::Set`
```suggestion
    msgs::Set(msg.mutable_step_size(), this->stepSize);
```

Reply:
updated to use `msgs::Set`. [497d8e1](https://github.com/gazebosim/gz-sim/pull/2340/commits/497d8e1253208c87213d01823d1c2764fa134c1b)